bundles/CoreBundle/src/Migrations/Version20230203160742.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230203160742 extends AbstractMigration
```

Comment:
Shouldn't it be a migration of the UuidBundle?

Reply:
Moved.